src/replication/replicator.rst

```diff
@@ -43,107 +48,151 @@ Let's say you POST the following document into ``_replicator``:
 
     {
         "_id": "my_rep",
-        "source":  "http://myserver.com:5984/foo",
+        "source":  "http://user:pass@localhost:5984/foo",
         "target":  "http://user:pass@localhost:5984/bar",
-        "create_target":  true
+        "create_target":  true,
+        "continuous": true
     }
 
 In the couch log you'll see 2 entries like these:
 
 .. code-block:: text
 
-    [Thu, 17 Feb 2011 19:43:59 GMT] [info] [<0.291.0>] Document `my_rep` triggered replication `c0ebe9256695ff083347cbf95f93e280+create_target`
-    [Thu, 17 Feb 2011 19:44:37 GMT] [info] [<0.124.0>] Replication `c0ebe9256695ff083347cbf95f93e280+create_target` finished (triggered by document `my_rep`)
-
-As soon as the replication is triggered, the document will be updated by
-CouchDB with 3 new fields:
+    [notice] 2017-04-05T17:16:19.646716Z node1@127.0.0.1 <0.29432.0> -------- Replication `"a81a78e822837e66df423d54279c15fe+continuous+create_target"` is using:
+        4 worker processes
+        a worker batch size of 500
+        20 HTTP connections
+        a connection timeout of 30000 milliseconds
+        10 retries per request
+        socket options are: [{keepalive,true},{nodelay,false}]
+    [notice] 2017-04-05T17:16:19.646759Z node1@127.0.0.1 <0.29432.0> -------- Document `my_rep` triggered replication `a81a78e822837e66df423d54279c15fe+continuous+create_target`
+
+Replication state of this document can then be queried from
```

Comment:
I am thinking of adding this

```
Replication document contents stay the same. Previously, before version
2.1, it was updated with the ``triggered`` state.
```

Above in `versionchanged` section there is mention of it as well, then we mention how the whole thing as well ```documents
added to it will trigger replications. Create (``PUT`` or ``POST``) a document
to start replication. ``DELETE`` a replication document to cancel an ongoing
replication.```